dockerclient/client.go

```diff
@@ -1050,7 +1051,7 @@ func (e *ClientExecutor) archiveFromContainer(from string, src, dst string) (io.
 		containerConfig = other.Container.Config
 	} else {
 		klog.V(5).Infof("Creating a container temporarily for image input from %q in %s", from, src)
-		_, err := e.LoadImage(from)
+		_, err := e.LoadImage(from, "")
```

Comment:
Do we need platform here as well ? I'm trying to avoid breaking public API where its not needed.

Reply:
This code path is called while we're handling a COPY instruction, so unless you're also aiming to add a `Platform` field to the `Copy` type, there won't be a value to pass here.

Reply:
ackd fixed.